runtime/protocol/http-client-engines/test-suite/proxy-scripts/fakeupstream.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
**Comment**: This file probably needs a copyright header just like our Kotlin files do.